components/hab/src/command/cli/setup.rs

```diff
@@ -148,15 +149,24 @@ pub fn start(ui: &mut UI, cache_path: &Path) -> Result<()> {
         let auth_token = prompt_auth_token(ui)?;
         write_cli_config_auth_token(&auth_token)?;
     } else {
-        ui.para("Okay, maybe another time.")?;
+        ui.para("Okay, maybe another time. You can also set a `HAB_AUTH_TOKEN` envvar or pass \
+                 the `--auth` flag to the cli!")?;
     }
+    ui.heading("Supervisor Control Gateway Secret")?;
+    ui.para("The Supervisor control gateway secret is used to authenticate hab client commands \
+             to a Supervisor. When a new Supervisor is created, a unique secret is generated \
+             for it automatically and stored locally in a file: `/hab/sup/default/CTL_SECRET`. \
+             When issuing commands to a local Supervisor, this file is read for the secret. \
+             Typically, a default Supervisor control gateway secret would only be needed if you \
+             with to send commands to a remote supervisor.")?;
```

Comment:
The tricky bit here is that if you want to set a value here, it really needs to be the secret being used by another Supervisor that you want to communicate with remotely.

I wish we had a better way to manage multiple Supervisors here :frowning:

Reply:
I agree with on both counts. Adjusted.